std/asys/io/IpcMessage.hx

```diff
@@ -0,0 +1,21 @@
+package asys.io;
+
+import asys.net.Socket;
+
+/**
+	A message sent over an IPC channel. Sent with `Process.send` to a sub-process
+	or with `CurrentProcess.send` to the parent process. Received with
+	`Process.messageSignal` from a sub-process, or `CurrentProcess.messageSignal`
+	from the parent process.
+**/
+typedef IpcMessage = {
```

Comment:
Why is it not parametrized?

Reply:
I goes through the Haxe serialiser, so it works through dynamic anyway.

Reply:
It's used in public API of `asys.Process`. More type safety wouldn't hurt :)